Dockerfile

```diff
@@ -36,11 +36,13 @@ RUN python /tmp/get-poetry.py --version $POETRY_VERSION
 # Configure poetry
 COPY poetry.toml /root/.config/pypoetry/config.toml
 
-# Install our python package dependencies
+# Install our python package and dependencies
 WORKDIR /build
-COPY pyproject.toml poetry.lock ./
+COPY pyproject.toml poetry.lock setup.py README.md ./
+COPY commands_base commands_base
 RUN . /root/.poetry/env \
- && poetry install
+ && poetry install \
+ && pip install .
```

Comment:
what is pip install doing for us that poetry install does not?

Reply:
+1

Reply:
So poetry is not a package deployment tool, when you do `poetry install` it does add your package to `site-packages` but as a symlink (the equivalent of `pip install -e`), the symlink is then broken because we are not leaving the source code in its original location (I don't think we should rely on that). What `pip install` here is doing is actually deploying/installing the commands base code 🙂 

More info here: https://github.com/python-poetry/poetry/issues/1301

Reply:
Why are we not leaving the source code in its original location?  And if we shouldn't rely on that to work (I'm not sure why we shouldn't), why are we calling both poetry install and pip install?  If we're using pip to install the package, shouldn't we be calling poetry install with `--no-root` so that we don't do it twice?

Reply:
Whoops!  Looks like you're doing that now.  Awesome! 

But yes, why are we not comfortable just installing with poetry?

Reply:
discussed offline, but basically poetry is installing in dev-mode which relies on install location, since this will be used as a base container we don't have a way to ensure that the install location will be preserved, installing using pip means the source code is copied into site-packages which is safer/cleaner